src/NodaTime/DateInterval.cs

```diff
@@ -134,6 +134,25 @@ public bool Contains(LocalDate date)
             return Start <= date && date <= End;
         }
 
+        /// <summary>
```

Comment:
Possibly make explicit mention that an interval contains itself?

Reply:
OK. Should I put this inside `remarks` or just on the main text?

Reply:
Either works for me :)